### DISCUSSION TITLE
alist安装在docker，怎么把docker外的局域网内其他主机的硬盘挂载上alist？

### DISCUSSION THREAD

**Discussion by freefly22** [AlistGo/alist#1039] [Q&A]
我的alist装在IP地址192.168.0.120的docker上，我要挂载的硬盘在192.168.0.4，对应的FTP服务器也开启了，在PC上能打开FTP。但是在alist上添加FTP，加不上去。求教，请说得详细点，感谢！
![image](https://user-images.githubusercontent.com/73732106/166890879-9093abef-e0c7-4613-a6d4-9f86c75514ea.png)

**Comment by xhofe** [comment]
https://docs.docker.com/network/
